src/main/java/com/autotune/analyzer/workerimpl/BulkJobManager.java

```diff
@@ -301,11 +377,77 @@ public void run() {
         }
     }
 
+    /**
+     *
+     * @param jobData Recommendation/Summary/Error data to be sent in the kafka topic
+     * @param experimentName name of the experiment
+     * @param topic kafka topic to which the data is sent
+     * @param failedAPI contains the name of the API that failed to construct the corresponding response (Metadata, Experiments, Recommendations)
+     * @return filtered JSON message to be sent to kafka topic
+     * @throws Exception exception if raised during filter process
+     */
+    private String buildKafkaResponse(BulkJobStatus jobData, String experimentName, String topic, String failedAPI) throws Exception {
```

Comment:
kafka Response should be configurable via config varaible

Reply:
Added new ENVs to make it configurable